proposals/0085-package-manager-command-name.md

```diff
@@ -103,7 +103,7 @@ updated the proposal to emphasize only `swift build`.
 We considered using `swift build` as the top level command for the package
 manager and moving the other verbs from being flags to being subcommands of
 `swift build`, instead of introducing a `package` command (e.g., `swift
-init`). We think this reads poorly and is less clear than making them `package`
+build init`). We think this reads poorly and is less clear than making them `package`
```

Comment:
Again, I am not entirely sure about this change, but I think the author meant to write 'swift build init' instead of 'swift init'.